src/Security/RateLimitingLogEntry.php

```diff
@@ -0,0 +1,115 @@
+<?php
+/**
+ * A model that represents a single rate limiting log entry.
+ *
+ * @package EightshiftForms\Security
+ */
+
+namespace EightshiftForms\Security;
+
+use RuntimeException;
+
+/**
+ * A model representing a single rate limiting log entry.
+ */
+class RateLimitingLogEntry {
+    public function __construct(
+        public readonly ?int $logId = null,
+        public readonly ?int $formId = null,
+        public readonly string $userToken,
+        public readonly string $activityType,
+        public readonly ?int $createdAt = null,
+    ) {}
```

Comment:
Required parameters after optional are deprecated since PHP 8.0

https://php.watch/versions/8.0/deprecate-required-param-after-optional

Reply:
Addressed in [6978bc9](https://github.com/infobipcth/eightshift-forms/pull/1/commits/6978bc9a064e27f6fcfe1bd2a8037b9c33572070)